### PR TITLE
Fix double `nk:NT` header in ImportActivity which cause 402 response.

### PR DESCRIPTION
**Pull request by leeeboo**
Fix double `nk:NT` header in ImportActivity which cause 402 response.

### CODE REVIEW

**Comment by RobertWojtowicz** [comment]
thank you very much, it works :)

**Comment by davidkroell** [comment]
Works for me as well, thank you

File: Activity.go [hunk 1]
```diff
@@ -170,7 +170,7 @@ func (c *Client) ImportActivity(file io.Reader, format ActivityFormat) (int, err
 		return 0, err
 	}
 
-	req.Header.Add("nk", "NT")
+	//req.Header.Add("nk", "NT")
```
Comment (abrander) [inline Activity.go:173]: Simply remove this line, history is preserved in git, if we ever need it again.

**Review by abrander — CHANGES_REQUESTED** [review]
This is an awesome catch and an embarrassing mistake on my part :-)

Thanks for fixing this. I believe this can be merged as soon as the minor detail is fixed.

**Comment by abrander** [comment]
Well, in the interest of moving forward, I'll merge this and fix it afterward. Thanks for your contribution, @leeeboo.

**Pull request merged**